### ISSUE TITLE
not working for ket

### ISSUE THREAD

**Issue by abhii-roy** [MizarZh/mathlive-in-editor-mode#3]
It's not working properly for `\ket{}`. After adding `\ket{}`, one is not able to add anything within `{}`

**Comment by MizarZh** [comment]
Can you provide a gif or video showing the problem? I can put something within `{}`.

![p](https://github.com/user-attachments/assets/09cad709-9c94-4ef7-bbce-9936fb15b36a)

**Comment by MizarZh** [comment]
Seems like a problem in mathlive. I've opened an issue. https://github.com/arnog/mathlive/issues/2488#issue-2467741079

It can be worked around by using customized macros. For example:
```json
mf.macros = { ... mf.macros,
    ket: '| {#1} \\langle'
}
```

This plugin doesn't have this function yet. I will work on this as soon as possible.

**Comment by abhii-roy** [comment]
Thanks. Meanwhile having a macro for this is an excellent idea.

**Comment by MizarZh** [comment]
I've just released 0.2.0 version. Macros still doesn't work to fix the `\ket` issue, but you can work around it by defining a new inline shortcut:
```js
{
ket: "|\\rangle"
}
```

Type in `ket`, then `|>` will show up.
